MaxText/configs/models/llama2-70b.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# model config for llama2-7b
+
+base_emb_dim: 4096
+base_num_query_heads: 64
+base_num_kv_heads: 8
+base_mlp_dim: 11008
+base_num_decoder_layers: 80
+head_dim: 128
+mlp_activations: ["silu","linear"]
+vocab_size: 32000
+enable_dropout: False
+vocab_relative_path: "tokenizer.llama2"
+logits_via_embedding: False
+normalization_layer_epsilon: 1.0e-5
+decoder_block: "llama2"
+attention: "dot_product"
```

Comment:
Please don't see the attention block or the enable_dropout flag -- users might want to override those. LGTM otherwise so giving you an approval.

Reply:
Done - thanks Rafi!